src/roiextractors/extractors/tiffimagingextractors/scanimagetiffimagingextractor.py

```diff
@@ -79,11 +80,30 @@ def __init__(
             When frames_per_slice > 1 (multiple frames per slice), this parameter specifies which frame to use
             for each slice. Must be between 0 and frames_per_slice-1. If None and frames_per_slice > 1,
             a ValueError will be raised.
+        load_series_as_contigious_samples : bool
+            False by default.
+
+            When true, all the samples are loaded as they appear in the acquisition cycle:
+
+            depth_1_frame_sample_1
```

Comment:
This isn't exactly what I had in mind. depth should still be a distinct dimension, it's just that the frames_per_slice are concatenated in time instead of being treated as different dimensions.

Reply:
Yeah, depth is still a different dimension it is just that your volumes are larger than they actually are. I re-wrote this explanation let me know if this matches what you had in mind.

Reply:
Wait, I pushed the wrong one, let me correct.

Reply:
Actually, easier to discuss it here:

![image](https://github.com/user-attachments/assets/45c1fe13-af5b-44ce-b10c-de18d15e8eb0)

This is a representation of the frames as acquired -with artificial volume- that can be used to implement the proposed workflow C here:

https://github.com/catalystneuro/neuroconv/issues/1260